fabulaws-config.yml

```diff
@@ -144,31 +144,40 @@
     web: [opendebates-sg, opendebates-web-sg]
 
 # Mapping of environment and role to EC2 instance types (sizes)
+# Except for very small instance types where it doesn't matter as much,
+# it's best to use 'c3' or 'm3' generation instances that have local SSD
+# storage for swap. This is especially true for 'web' instances that will
+# be used to create AMIs (and keep a small volume_size), as it dramatically
+# speeds up AMI creation (by ~10 min).
+# Additionally, 2xlarge or higher instance types are needed to get >= 1000 Mbps
+# on the network. This is important for the cache server which might not have
+# a lot of CPU load but will be transfering lots of data back & forth. See:
+# http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-ec2-config.html
   instance_types:
     testing:
-      cache: c4.large
-      db-master: m4.2xlarge
-      db-slave: m4.2xlarge
-      web: c4.xlarge
-      worker: m4.large
+      cache: c3.2xlarge # for 1000Mbps network
+      db-master: c3.2xlarge
+      db-slave: c3.2xlarge
```

Comment:
I still don't have a great understanding of the nuances of different instance types, but does it matter that the c3 are not EBS optimized while the m4 were. I believe that's important for DB performance?